### ISSUE TITLE
[live555] update to 2024.06.26

### ISSUE THREAD

**Issue by spacepirate0001** [microsoft/vcpkg#40118]
### Operating system

Linux

### Compiler

Clang

### Steps to reproduce the behavior

```Shell
'./vcpkg install live555'
```

### Failure logs

#17 1.672 Computing installation plan...
#17 1.698 The following packages will be built and installed:
#17 1.698     live555:x64-linux@2024-05-15
#17 1.698 Detecting compiler hash for triplet x64-linux...
#17 1.992 Compiler found: /usr/bin/c++
#17 1.997 Restored 0 package(s) from /root/.cache/vcpkg/archives in 13.8 us. Use --debug to see more details.
#17 1.997 Installing 1/1 live555:x64-linux@2024-05-15...
#17 1.997 Building live555:x64-linux@2024-05-15...
#17 2.009 -- Downloading http://live555.com/liveMedia/public/live.2024.05.15.tar.gz -> live.2024.05.15.tar.gz...
#17 2.664 [DEBUG] To include the environment variables in debug output, pass --debug-env
#17 2.664 [DEBUG] Trying to load bundleconfig from /vcpkg/vcpkg-bundle.json
#17 2.664 [DEBUG] Failed to open: /vcpkg/vcpkg-bundle.json
#17 2.664 [DEBUG] Bundle config: readonly=false, usegitregistry=false, embeddedsha=nullopt, deployment=Git, vsversion=nullopt
#17 2.664 [DEBUG] Metrics enabled.
#17 2.664 [DEBUG] Feature flag 'binarycaching' unset
#17 2.664 [DEBUG] Feature flag 'compilertracking' unset
#17 2.664 [DEBUG] Feature flag 'registries' unset
#17 2.664 [DEBUG] Feature flag 'versions' unset
#17 2.664 [DEBUG] Feature flag 'dependencygraph' unset
#17 2.664 [DEBUG] Detected docker in cgroup
#17 2.664 [DEBUG] 1000: execute_process(curl --fail -L http://live555.com/liveMedia/public/live.2024.05.15.tar.gz --create-dirs --output /vcpkg/downloads/live.2024.05.15.tar.gz.102.part)
#17 2.664 [DEBUG] 1000: cmd_execute_and_stream_data() returned 22 after   128163 us
#17 2.664 error: Missing live.2024.05.15.tar.gz and downloads are blocked by x-block-origin.
#17 2.664 error: http://live555.com/liveMedia/public/live.2024.05.15.tar.gz: curl failed to download with exit code 22
#17 2.664   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
#17 2.664                                  Dload  Upload   Total   Spent    Left  Speed
#17 2.664
#17 2.664 curl: (22) The requested URL returned error: 404 Not Found
#17 2.664
#17 2.664 [DEBUG] /source/src/vcpkg/base/downloads.cpp(1030):
#17 2.664 [DEBUG] Time in subprocesses: 128163us
#17 2.664 [DEBUG] Time in parsing JSON: 3us
#17 2.664 [DEBUG] Time in JSON reader: 0us
#17 2.664 [DEBUG] Time in filesystem: 416us
#17 2.664 [DEBUG] Time in loading ports: 0us
#17 2.664 [DEBUG] Exiting after 129 ms (128646us)
#17 2.664
#17 2.664 CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:32 (message):
#17 2.664
#17 2.664       Failed to download file with error: 1
#17 2.664       If you are using a proxy, please check your proxy setting. Possible causes are:
#17 2.664
#17 2.664       1. You are actually using an HTTP proxy, but setting HTTPS_PROXY variable
#17 2.664          to `https://address:port`. This is not correct, because `https://` prefix
#17 2.664          claims the proxy is an HTTPS proxy, while your proxy (v2ray, shadowsocksr
#17 2.664          , etc..) is an HTTP proxy. Try setting `http://address:port` to both
#17 2.664          HTTP_PROXY and HTTPS_PROXY instead.
#17 2.664
#17 2.664       2. If you are using Windows, vcpkg will automatically use your Windows IE Proxy Settings
#17 2.664          set by your proxy software. See https://github.com/microsoft/vcpkg-tool/pull/77
#17 2.664          The value set by your proxy might be wrong, or have same `https://` prefix issue.
#17 2.664
#17 2.664       3. Your proxy's remote server is out of service.
#17 2.664
#17 2.664       If you've tried directly download the link, and believe this is not a temporary
#17 2.664       download server failure, please submit an issue at https://github.com/Microsoft/vcpkg/issues
#17 2.664       to report this upstream download server failure.
#17 2.664
#17 2.664
#17 2.664 Call Stack (most recent call first):
#17 2.664   scripts/cmake/vcpkg_download_distfile.cmake:270 (z_vcpkg_download_distfile_show_proxy_and_fail)
#17 2.664   ports/live555/portfile.cmake:3 (vcpkg_download_distfile)
#17 2.664   scripts/ports.cmake:192 (include)
#17 2.664
#17 2.664
#17 2.667 error: building live555:x64-linux failed with: BUILD_FAILEDSee https://learn.microsoft.com/vcpkg/troubleshoot/build-failures?WT.mc_id=vcpkg_inproduct_cli for more information.
#17 2.667 Elapsed time to handle live555:x64-linux: 670 ms
#17 2.669 Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
#17 2.669 Then check for known issues at:
#17 2.669   https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+live555
#17 2.669 You can submit a new issue at:
#17 2.669   https://github.com/microsoft/vcpkg/issues/new?title=[live555]+Build+error+on+x64-linux&body=Copy+issue+body+from+%2Fvcpkg%2Finstalled%2Fvcpkg%2Fissue_body.md
#17 2.669
#17 ERROR: process "/bin/sh -c apt-get install libssl-dev && cd /vcpkg && ./vcpkg install live555" did not complete successfully: exit code: 1
